### DISCUSSION TITLE
Announcing Monthly 'Meet an Architect' Meetings: First One on November 5, 2020

### DISCUSSION THREAD

**Discussion by johnbent** [Seagate/cortx#437] [Show and tell] [answered]
All,

We are very happy to announce that we are starting a monthly CORTX 'Meet an Architect' meeting in which a CORTX architect will present a preparation of interest and in which we will have a follow-up Q&A session.  Our first 'Meet an Architect' meeting will be on Thursday, November 5, 2020 in which @nikitadanilov will present about the core motr architecture.

Hope to see you there!  

[Calendar Invite Link](https://raw.githubusercontent.com/Seagate/cortx/main/doc/meetings/CORTX_Monthly_Meet_an_Architect_Series.ics)

For more info, please visit our [Monthly Meetings webpage](https://github.com/Seagate/cortx/blob/main/doc/meetings/README.rst).

**Comment by justinzw** [comment, marked as answer]
I'm very excited about this! If anyone has any suggestions on topics they would like us to include please suggest them here too!

For me I'd love to hear about - "How CORTX is optimized for mass capacity storage devices?"
